packages/gitbook/src/components/Header/Dropdown.tsx

```diff
@@ -111,31 +111,38 @@ export function DropdownMenu(props: { children: React.ReactNode }) {
  * Menu item in a dropdown.
  */
 export function DropdownMenuItem(props: {
-    href: string;
+    href: string | null;
     active?: boolean;
+    className?: ClassValue;
     children: React.ReactNode;
 }) {
-    const { children, active = false, href } = props;
+    const { children, active = false, href, className } = props;
+
+    if (href) {
+        return (
+            <Link
+                href={href}
+                prefetch={false}
+                className={tcls(
```

Comment:
Curious to know why we went for `tcls` instead of `clsx` on GBO?

Reply:
It's a mistake, but it's like that from the beginning. At some point we should change it to improve perfs.